test/java/org/opendatakit/briefcase/reused/UncheckedFilesTest.java

```diff
@@ -0,0 +1,65 @@
+package org.opendatakit.briefcase.reused;
+
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.opendatakit.briefcase.reused.UncheckedFiles.createFile;
+import static org.opendatakit.briefcase.reused.UncheckedFiles.createTempDirectory;
+import static org.opendatakit.briefcase.reused.UncheckedFiles.deleteRecursive;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UncheckedIOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+
+public class UncheckedFilesTest {
+  private static Path tempDir;
+  private Path temp;
+
+
+  @Before
+  public void setUp() {
+    tempDir = createTempDirectory("briefcase");
+    temp = Paths.get(tempDir.toString() + "/test.txt");
+    createFile(temp);
+  }
+
+  @After
+  public void tearDown() {
+    deleteRecursive(tempDir);
+  }
+
+  @Test(expected = UncheckedIOException.class)
+  public void newInputStream_should_throw_exception() {
+    UncheckedFiles.delete(temp);
+    UncheckedFiles.newInputStream(Paths.get(tempDir.toString() + "/test.txt"));
+  }
+
+  @Test
+  public void closeInputStream_should_handle_null() {
+    UncheckedFiles.closeInputStream(null);
+  }
+
+  @Test(expected = IOException.class)
+  public void closeInputStream_should_close() throws IOException {
+    InputStream inputStream = UncheckedFiles.newInputStream(Paths.get(tempDir.toString() + "/test.txt"));
+    // close inputStream
+    UncheckedFiles.closeInputStream(inputStream);
+
+    // after close, .available() should throw exception
+    inputStream.available();
+  }
+
+  @Test(expected = UncheckedIOException.class)
+  public void closeInputStream_should_throw_exception() throws IOException {
+    InputStream inputStream = mock(InputStream.class);
+    doThrow(new IOException()).when(inputStream).close();
+
+    UncheckedFiles.closeInputStream(inputStream);
+  }
```

Comment:
I don't think that this test justifies adding Mockito to the project. I know it's ugly, but the suggested change would take care of the test without Mockito.

```suggestion
  @Test(expected = UncheckedIOException.class)
  public void closeInputStream_should_throw_exception() throws IOException {
    InputStream is = new InputStream() {
      @Override
      public int read() throws IOException {
        return 0;
      }

      @Override
      public void close() throws IOException {
        throw new IOException("chuchu");
      }
    };
    UncheckedFiles.closeInputStream(is);
  }
```

Reply:
Sure, my thinking was that mockito could be introduced and used in other tests to help cover exception handling or other cases. 

Maybe it can be introduced when a pull request actually contains moderate usage of mockito.